player/command.c

```diff
@@ -4249,6 +4262,7 @@ static const struct property_osd_display {
     {"sub", "Subtitles"},
     {"secondary-sid", "Secondary subtitles"},
     {"sub-pos", "Sub position"},
+    {"secondary-sub-pos", "Secondary Sub position"},
```

Comment:
```suggestion
    {"secondary-sub-pos", "Secondary sub position"},
```
to be consistent with `"Secondary sub delay"`